thewalrus/_hafnian.py

```diff
@@ -817,23 +775,25 @@ def hafnian(
         return hafnian_approx(A, num_samples=num_samples)
 
     if loop:
+        if method == "recursive":
+            warnings.warn("Recursive algorithm does not support the loop hafnian")
         return loop_hafnian(A, D=None, reps=None, glynn=glynn)
 
+    if method == "recursive":
+        return recursive_hafnian(A)
+
```

Comment:
here you need to be careful, if `method="glynn"` then in the next line `glynn==True` otherwise, if `method="exclincl"` then in the line below `glynn=False"

Reply:
Isn't it what we wrote at lines 721-725?

Reply:
I just saw lines 722 - 727. I think now I get it. You can ignore this comment

Reply:
Yup apologies. You are absolutely right. It is now just amatter of removing the `int()` and `/` in the `solve` function and do a second benchmark and you are good to go.